tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -2175,10 +2175,20 @@ def create_float_array_tuple_ref(a : 'int[:,:]'):
         b = (a[0,:], a[1,:])
         c = array(b, dtype=float)
         return c
+    def create_bool_array_tuple_ref(a : 'int[:,:]'):
+        from numpy import array
+        b = (a[0,:], a[1,:])
+        c = array(b, dtype=float)
```

Comment:
Should this not be an array of `bool`?
```suggestion
        c = array(b, dtype=bool)
```

Reply:
Oops you are right. The skip is also no longer necessary here. Fixed ff77793f6